vulcan/lib/client/jsx/components/workflow/manager.jsx

```diff
@@ -0,0 +1,86 @@
+// Framework libraries.
```

Comment:
@dtm2451 , I think this is an example component you can build off of and take a look at. You'll want to clean up the rendered stuff and break that into multiple child components, but this should work and give you an idea of how data flows and how to manipulate it. We can talk through it in detail when you have time and have fiddled with it.